Toolkit concept docs XAML: batch 1 (#554)
Esri/arcgis-maps-sdk-dotnet-toolkit: docs/compass.md

# Compass

> Show a compass direction for a `GeoView`. Resets the view orientation when tapped/clicked.

Reviewer: I would keep it explicitly mapview/sceneview for clarity

- Can be configured to hide itself automatically when the heading is 0 via the `AutoHide` property.
- Allows display of a manually-set heading via the `Heading` property.

## Usage

### .NET MAUI:

```xml
<Grid>
    <esri:MapView x:Name="mapView" />
    <toolkit:Compass Margin="15"
                     AutoHide="False"
                     GeoView="{x:Reference mapView}"
                     HeightRequest="50"
                     WidthRequest="50" />
</Grid>
```

### UWP/WinUI:

```xml
<Grid>
    <esri:MapView x:Name="mapView" />
    <toolkit:Compass Width="50"
                     Height="50"
                     Margin="15"
                     AutoHide="False"
                     GeoView="{Binding ElementName=mapView}" />
</Grid>
```

### WPF:

> The usage in WPF is identical to UWP/WinUI minus one important distinction. The `Compass` should be accessed with the same prefix as the `GeoView`. 

Reviewer 2: ```suggestion
The usage in WPF is identical to UWP/WinUI with one important distinction: `Compass` is accessed with the same namespace prefix as `GeoView`.
```
